package.json

```diff
@@ -6,8 +6,8 @@
     "ng": "nx",
     "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main",
     "start": "nx serve",
-    "start:api": "nx serve snapodds-api",
-    "build": "nx run-many --target=build --projects=snapodds-builder,snapodds",
+    "start:demo": "nx build snapodds-demo && nx serve snapodds-demo",
+    "build": "nx run-many --target=build --projects=snapodds-builder,snapodds,snapodds-demo",
```

Comment:
build task is intended to be used for building & publishing the artifacts, as you are not publishing snapoods-demo, why do you want to build it here?